typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,16 @@
+const fullName: string = 'Lucas Yasumura';
+const isCool: boolean = true;
+const totalPets: number = 1;
+const empty: null = null;
+const nothing: undefined = undefined;
```

Comment:
Rather than assigning a value of `undefined` directly, you should just create the `nothing` variable without assigning it a value. If you do not give a variable a value, it will already have a value of `undefined`